profiles/adb.jsonld

```diff
@@ -0,0 +1,231 @@
+{
+  "@context": "http://xapi.vocab.pub/profiles/profile-context.jsonld",
+  "id": "https://w3id.org/xapi/adb",
+  "type": "Profile",
+  "conformsTo": "https://github.com/DataInteroperability/xapi-profiles/tree/master#1.0-development",
+  "prefLabel": {
+  	"en": "Actionable Data Book (ADB) Profile"
+  },
+  "definition": {
+    "en": "The xAPI ABD profile is used to identify interactions between an actor and an eBook activity based on the IEEE ADB project."
+  },
+  "seeAlso": "https://standards.ieee.org/develop/indconn/adb/",
+  "author": {
+    "type": "Group",
+    "name": "IEEE Industry Connections Actionable Data Book (ADB) Team"
+  },
+  "versions": [
+    {
+      "id": "https://w3id.org/xapi/adb/v1",
+      "generatedAtTime": "2017-06-30T8:26:00Z",
+      "wasRevisionOf": ["https://w3id.org/xapi/adb"]
+    }
+  ],
+  "concepts": [
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/annotated",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "definition": {
+        "en": "Provide notes or symbols for further explanation or thoughts while reading."
+      },
+      "narrower": [ {
+          "id": "https://w3id.org/xapi/adb/verbs/highlighted"
+      } ],
+      "relatedMatch": [ {
+          "id": "https://w3id.org/xapi/acrossx/verbs/annotated"
+      } ],
+      	"prefLabel": {
+        "en": "annotated"
+      }
+    },
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/arrived",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "broadMatch": {
+        "id": "http://adlnet.gov/expapi/verbs/progressed"
+      },
+      "definition": {
+        "en": "Indicates the actor arrived a specific location by movement or progress. "
+      },
+      "scopeNote": {
+        "en": "To arrive a specific scene within a story or book experience."
```

Comment:
Also, the contents of scopeNote as used aren't about scope, they're about what the verb means, which should go in definition anyways